.github/workflows/test_flask.yml

```diff
@@ -0,0 +1,16 @@
+name: Automated testing
```

Comment:
If we give this a different name (e.g. Flask Tests) it will be clearer which test is which when we look at the checks on Github.